alfresco-transform-core-aio/alfresco-transform-core-aio/src/main/java/org/alfresco/transformer/transformers/ImageMagickAdapter.java

```diff
@@ -59,10 +60,10 @@
     private static String ID = "imagemagick";
     private ImageMagickCommandExecutor commandExecutor;
 
-    public ImageMagickAdapter() throws Exception 
+    public ImageMagickAdapter(GlobalProperties.ImageMagick execPaths) throws Exception

```

Comment:
It would be simpler to just pass the 3 strings in, rather than having to worry about another class, especially as this class is external.